pkgs/misc/preload/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, autoconf, automake, pkg-config, bash, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "preload";
+  version = "0.6.4";
+
+  src = fetchurl {
+    url =
+      "http://downloads.sourceforge.net/sourceforge/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-0KVY6DyymlHZ2Wc27zn0tOVeQ6WJrRrsWUoEjKIvgWs=";
+  };
+
+  patches = [ ./Makefile.patch ];
+
+  nativeBuildInputs = [ autoconf automake pkg-config ];
+  buildInputs = [ bash glib ];
+
+  configureFlags = [
+    "--localstatedir=/var"
+  ];
+
+  postInstall = ''
+    make sysconfigdir=/$out/etc/conf.d install
+  '';
+
+  meta = with lib; {
+    description =
+      "Makes applications run faster by prefetching binaries and shared objects";
+    homepage = "http://sourceforge.net/projects/preload";
+    license = licenses.gpl2;
```

Comment:
`gpl2` is deprecated, use `gpl2Only` or `gpl2Plus`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206